components/resources/quizzes/shared/BluePrint.tsx

```diff
@@ -40,7 +39,9 @@ const BluePrint = ({ onNextClick, onBackClick }: BluePrintProps) => {
       <div className="grid py-6 place-items-center">
         <Button
           label="View results"
-          onClick={onNextClick}
+          onClick={() => {
```

Comment:
there's no need for this file to be changed at all in your PR. I would just remove the formatting change if possible to make your PR simpler